frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-section-card-content/common-section-card-content.component.html

```diff
@@ -1,14 +1,14 @@
 <div class="content-section">
-  <div *ngFor="let obj of data">
-    <div *ngFor="let item of obj | keyvalue">
-      <span style="font-weight: bold">{{ item.key }}:</span>
-      <ng-container *ngIf="item.key !== 'tags'; else chipSection">
-        <span>{{ item.value }}</span>
+  <div *ngFor="let key of objectKeys(data)">
```

Comment:
Please use `keyvalue` pipe, then we don't need extra `objectKeys` function.

<div *ngFor="let item of data | keyvalue">
  <div class='item-key-value'>
    <span class="ft-16-600 item-key">{{ item.key }}:</span>
    <ng-container *ngIf="item.key !== 'Tags'; else chipSection">
      <span class="ft-16-500 item-value">{{ item.value }}</span>
    </ng-container>

    <ng-template #chipSection>
      <mat-chip-list>
        <mat-chip *ngFor="let tag of item.value" class="chip">{{ tag }}</mat-chip>
      </mat-chip-list>
    </ng-template>
  </div>
</div>

Reply:
can you provide more info here? didn't get it what's this is for `{{ item.key }}: {{ item.value }}`

Reply:
weird, I edited it, I must have pasted it into the formatting in a bizarre way

Reply:
ok, that worked but it is showing all keys in an alphabetically sorted manner, not as given in the object according to UI. Is that fine?

Reply:
ah okay, let me reverse then, because maybe then we do want an array. So I think it made sense for Khanjan to use an array for the Input, but we can just leave it with object.keys if it's working, I don't want to spend more time on it

Reply:
I found a solution if I do it like this
`<div *ngFor="let item of data | keyvalue: noSort">`
where noSort is
`noSort() {
    return 0;
  }`
  then it works fine but can pass only func there can't direct put 0 next to keyValue